ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpWork.java

```diff
@@ -207,4 +217,20 @@ public ReplicationMetricCollector getMetricCollector() {
   public void setMetricCollector(ReplicationMetricCollector metricCollector) {
     this.metricCollector = metricCollector;
   }
+
+  public ReplicationSpec getReplicationSpec() {
+    return replicationSpec;
+  }
+
+  public void setReplicationSpec(ReplicationSpec replicationSpec) {
+    this.replicationSpec = replicationSpec;
+  }
+
+  public FileList getFileList(Path backingFile, int cacheSize, HiveConf conf, boolean b) throws IOException {
```

Comment:
If this is used for tests, you can just mock the ReplDumpWork and return expected values.

Reply:
This is required for some old test which isn't  mockito based .

Reply:
removed the change altogether